modules/Bio/EnsEMBL/Compara/PipeConfig/OrthologQM_Alignment_conf.pm

```diff
@@ -288,24 +298,43 @@ sub pipeline_analyses {
             -module     => 'Bio::EnsEMBL::Compara::RunnableDB::OrthologQM::CalculateWGACoverage',
             -hive_capacity => 30,
             -batch_size => 10,
-            -parameters => { pipeline_url => $self->pipeline_url },
+            -parameters => { alignment_db => $self->pipeline_url },
             -flow_into  => {
-                '1'  => [ '?table_name=ortholog_quality' ],
-                '2'  => [ 'assign_wga_coverage_score' ],
+                3 => [ '?table_name=ortholog_quality' ],
+                2 => [ 'assign_wga_coverage_score' ],
             },
             -rc_name => '2Gb_job',
         },
 
         {   -logic_name => 'assign_wga_coverage_score',
             -module     => 'Bio::EnsEMBL::Compara::RunnableDB::OrthologQM::AssignQualityScore',
-            -hive_capacity     => 30,
-            -batch_size        => 10,
+            -parameters => {
+                'hashed_mlss_id' => '#expr(dir_revhash(#orth_mlss_id#))expr#',
+                'output_file'    => '#wga_dumps_dir#/#hashed_mlss_id#/#orth_mlss_id#.#member_type#.wga.tsv',
+            },
+            -hive_capacity     => 400,
         },
 
         {   -logic_name => 'reuse_wga_score',
             -module     => 'Bio::EnsEMBL::Compara::RunnableDB::OrthologQM::ReuseWGAScore',
-            -hive_capacity     => 30,
-            -batch_size        => 10,
+            -parameters => {
+                'homology_dumps_dir'        => $self->o('homology_dumps_dir'),
+                'hashed_mlss_id'            => '#expr(dir_revhash(#orth_mlss_id#))expr#',
+                'previous_wga_file'         => '#prev_wga_dumps_dir#/#hashed_mlss_id#/#orth_mlss_id#.#member_type#.wga.tsv',
+                'homology_mapping_flatfile' => '#homology_dumps_dir#/#hashed_mlss_id#/#orth_mlss_id#.#member_type#.homology_id_map.tsv',
+                'output_file'               => '#wga_dumps_dir#/#hashed_mlss_id#/#orth_mlss_id#.#member_type#.wga.tsv',
+            },
+            -hive_capacity     => 400,
+        },
+
+        {   -logic_name => 'copy_files_to_shared_loc',
+            -module     => 'Bio::EnsEMBL::Hive::RunnableDB::SystemCmd',
+            -parameters => {
+                'homology_dumps_dir'        => $self->o('homology_dumps_dir'),
+                'homology_dumps_shared_dir' => $self->o('homology_dumps_shared_dir'),
+                'cmd'                       => 'become #shared_user# /bin/bash -c "mkdir -p #homology_dumps_shared_dir# && rsync -rt #wga_dumps_dir#/ #homology_dumps_shared_dir#"',
```

Comment:
This is not correct - I want to copy to the files to the `homology_dumps_shared_dir`, which should *always* point at the `compara_ensembl` shared space. `homology_dumps_dir` points to the location of the homology dumps files, which could be the protein trees `pipeline_dir`. Since this has confused you twice, I will add comments to the params to clarify their purpose.